packages/wdio-allure-reporter/src/index.js

```diff
@@ -102,10 +102,10 @@ class AllureReporter extends WDIOReporter {
 
         if (!this.isMultiremote) {
             const { browserName, deviceName } = this.capabilities
```

Comment:
`const { browserName, device, deviceName } = this.capabilities`

Reply:
then use `device` instead of `this.capabilities.device`

Reply:
Good point!

Reply:
@mgrybyk or @christian-bromann Do you have a preference? We are using both the deconstructed variable as well as the `this.capabilities` in this code block...

Reply:
You may put device to deconstructed variables especially if you need it more than once